ivy/functional/ivy/manipulation.py

```diff
@@ -471,7 +471,7 @@ def reshape(
     }
 
     """
-    return current_backend(x).reshape(x, shape, copy, out=out)
+    return current_backend(x).reshape(x, shape, copy=copy)
```

Comment:
Hi @AnnaTz, the out argument should be left here. The @handle_out_argument decorator at the start of the function handles this functionality, and it's good to have it in ivy even if the backends do not support it in the same way! :)

Reply:
you can leave copy=copy though :)

Reply:
Great, it's been resolved then :)

Reply:
Alright, I have merged the master into the PR, so the reshape function is now as you suggested :)